Mathlib/NumberTheory/Padics/Complex.lean

```diff
@@ -0,0 +1,197 @@
+/-
+Copyright (c) 2025 María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: María Inés de Frutos-Fernández
+-/
+import Mathlib.Analysis.Normed.Unbundled.SpectralNorm
+import Mathlib.NumberTheory.Padics.PadicNumbers
+import Mathlib.Topology.Algebra.Valued.NormedValued
+import Mathlib.Topology.Algebra.Valued.ValuedField
+
+/-!
+# The field `ℂ_[p]` of `p`-adic complex numbers.
+
+In this file we define the field `ℂ_[p]` of `p`-adic complex numbers as the `p`-adic completion of
+an algebraic closure of `ℚ_[p]`. We endow `ℂ_[p]` with both a normed field and a valued field
+structure, induced by the unique extension of the `p`-adic norm to `ℂ_[p]`.
+
+## Main Definitions
+* `PadicAlgCl p` : the algebraic closure of `ℚ_[p]`.
+* `PadicComplex p` : the type of `p`-adic complex numbers, denoted by `ℂ_[p]`.
+* `PadicComplexInt p` : the ring of integers of `ℂ_[p]`.
+
+## Main Results
+
+* `PadicComplex.norm_extends` : the norm on `ℂ_[p]` extends the norm on `PadicAlgCl p`, and hence
+  the norm on `ℚ_[p]`.
+* `PadicComplex.isNonarchimedean` : The norm on `ℂ_[p]` is nonarchimedean.
+
+## Notation
+
+We introduce the notation `ℂ_[p]` for the `p`-adic complex numbers, and `𝓞_ℂ_[p]` for its ring of
+integers.
+
+## Tags
+
+p-adic, p adic, padic, norm, valuation, Cauchy, completion, p-adic completion
+-/
+
+noncomputable section
+
+open Valuation
+
+open scoped NNReal
+
+variable (p : ℕ) [hp : Fact (Nat.Prime p)]
+
+/-- `PadicAlgCl p` is the algebraic closure of `ℚ_[p]`. -/
```

Comment:
```suggestion
/-- `PadicAlgCl p` is a fixed algebraic closure of `ℚ_[p]`. -/
```
I think we should say it like this, because some people are keen to point out that because algebraic closures have many automorphisms, they're not unique up to unique isomorphism, so we should rather talk about "an" algebraic closure instead of "the" algebraic closure.